Composer/package.json

```diff
@@ -20,7 +20,9 @@
     "url-parse": "^1.5.1",
     "underscore": "^1.12.1",
     "dns-packet": "^1.3.4",
-    "json-ptr": "^2.1"
+    "json-ptr": "^2.1",
```

Comment:
Are these dependencies necessary? They seem out of scope for this PR.

Reply:
They were causing a security alert, but I've now handled that in a separate PR which has merged already (so these can/will go away on a rebase).